src/views/Picklists.vue

```diff
@@ -50,13 +55,31 @@ export default defineComponent({
   },
   computed: {
     ...mapGetters({
-      picklists: 'picklist/getPicklists'
+      picklists: 'picklist/getPicklists',
+      currentFacilityId: 'user/getCurrentFacility',
+      isScrollable: 'picklist/isScrollable'
     })
   },
+  methods: {
+    async getPickLists(vSize?: any, vIndex?: any) {
+      const viewSize = vSize ? vSize : process.env.VUE_APP_VIEW_SIZE;
+      const viewIndex = vIndex ? vIndex : 0;
+
+      this.store.dispatch('picklist/findPickList', { viewSize, viewIndex }).catch(err =>
+        this.store.dispatch('picklist/clearPicklist')
```

Comment:
I think we don't need to clear picklist on error, because if on first call api throws error then it will work fine, but if we get an error in infinite scroll call, then also we will be in catch block and all the picklist will get cleared.

Reply:
Updated sir